coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -103,6 +123,35 @@ def _parse_documentation_with_symbols(self,
             section is a named tuple of either ``Description``, ``Parameter``,
             ``ExceptionValue`` or ``ReturnValue``.
         """
+
+        def startswith(line, starter):
```

Comment:
@jayvdb and @userzimmermann  What may be a proper location for the function `startswith`?